src/en/glossary.md

```diff
@@ -39,26 +36,24 @@ following format and restrictions:
 
 **Endpoint** - The combination of a service name and a relation name.
 
-**Environment** - An Environment is a configured location where Services can
-be deployed onto. An Environment typically has a name, which can usually be
-omitted when there's a single Environment configured, or when a default is
-explicitly defined. Depending on the type of Environment, it may have to be
-bootstrapped before interactions with it may take place (e.g. EC2). The local
-environment configuration is defined in the ~/.juju/environments.yaml file.
+**Environment** - A configured location where services can be deployed onto. An
+Environment typically has a name, which can usually be omitted when there's a
+single Environment configured, or when a default is explicitly defined.
+Depending on the type of Environment, it may have to be bootstrapped before
+interactions with it may take place (e.g. EC2). The local environment
+configuration is defined in the ~/.juju/environments.yaml file.
```

Comment:
Sure, I think it was in the additions matt made a few days back, but as we are checking it over... I think it would be okay to simply say the environments.yaml file and maybe link to whereits location is explained